packages/ui/src/routes/bridge/BridgeConfirmPage.tsx

```diff
@@ -411,7 +453,10 @@ const BridgeConfirmPage: FunctionComponent<{}> = () => {
                                 : !quote ||
                                   isGasLoading ||
                                   isFetchingParams ||
-                                  isBridging
+                                  isBridging ||
+                                  (isBridgingNativeToken
+                                      ? false
```

Comment:
no need for `isBridgingNativeToken` condition. The hook should not fire the process if the user is bridging a native asset

Reply:
Similar to the previous thread. Let's continue the discussion there